pkg/sql/walk.go

```diff
@@ -86,7 +86,7 @@ func (v *planVisitor) visit(plan planNode) {
 	}
 
 	name := nodeName(plan)
-	var recurse bool
+	recurse := true
```

Comment:
I'm not quite sure what you mean, it's four lines down in its own conditional block. Where else should it go?